android/src/main/java/com/swmansion/rnscreens/CustomToolbar.kt

```diff
@@ -55,6 +75,15 @@ open class CustomToolbar(
 
     override fun requestLayout() {
         super.requestLayout()
+
+        val maybeAppBarLayout = parent as? CustomAppBarLayout
+        maybeAppBarLayout?.let {
+            if (shouldApplyLayoutCorrectionForTopInset && !it.isInLayout) {
+                it.onToolbarLayout(paddingTop)
```

Comment:
Reading this I have a problem with naming here. We call `onToolbarLayout` in `requestLayout`. This clearly contradicts. 

Why do we call this callback in `requestLayout`? Shouldn't it be called in `onLayout`?